_specs/ecip-proto-treasury-system.md

```diff
@@ -0,0 +1,450 @@
+
+To realise this ambitious vision, innovation must be encouraged and the ETC network must be consistently maintained and continuously improved. Blockchain protocols are highly complex distributed systems that are proven to be expensive to research, develop, and implement while operating within an  increasingly competitive developer market. ETC needs to position itself as a desirable platform for innovation where developers want to invest their time making improvements and building new applications.
+
+ETC can gain significant competitive advantage in the medium and long-term outlook by delivering a stable, decentralized and community-driven funding system for protocol development that is backed by a clear technical roadmap. To achieve this goal, stable and reliable funding is required, under the transparent and verifiable control of the Ethereum Classic community, through the ECIP process. With full transparency, proposals can be thoroughly questioned and rigorously examined by fellow community members, with community members themselves making collective funding decisions.
+
+ETC needs a vision to inspire developers - particularly those currently outside the ETC ecosystem, to build on the platform, ahead of the many other well-funded competitor platforms in the space. This treasury system is designed to do this quickly and efficiently and having a high value ETC will also mitigate against potential 51% attacks.
+
+We recognise this proposal outlines a significant change to the future governance of ETC. For this reason, we are proposing an element of built in obsolescence so the community has the opportunity to directly assess the value of the treasury system and in time, actively vote for its continuation and further development or indeed to allow it to be reversed.
+
+This proposal is not designed to deliver an end-state system, but a stepping stone solution on the path to the launch of a fully decentralized treasury system.
+
+
+### Specification
+
+#### Solution Overview and Rationale 
+
+This proposal opts for the most simple solution for the p-ECTS to encourage collaboration and transparency for this first-step treasury system.
+
+This specification represents the p-ECTS only, and a best-in-class treasury and governance model, such as outlined in [[1]](https://eprint.iacr.org/2018/435.pdf
+), can be proposed by the community via the ECIP mechanism in the future.
+
+We would also put forward that this treasury system would go through a formal verification process to ensure that it behaves as per the final specification.
+
+#### Parameters
+
+| Name | Description | Proposed value |
+|---|---|---|
+|**FORK_BLKNUM**|Block on which the Hard Fork will take place.|TBD|
+|**Initial client members addresses**|Collections of all MemberType.Client.|TBD|
+|**gitcoinAddress**|Gitcoin grant address.|TBD|
```

Comment:
Who controls this address?

Reply:
> Draft. https://github.com/input-output-hk/ECIPs/blob/ecip-proto-treasury/_specs/ecip-proto-treasury-system.md
> Discussion. #350 | [input-output-hk#2](https://github.com/input-output-hk/ECIPs/issuFYIes/2)

FYI, this draft link is dead.

Reply:
Good question, could this be where we setup the DAO that was being discussed in the #communityFund channel? With the distribution method via Gitcoin?